### PR TITLE
Fix status and peer list rending issues

### LINKED ISSUE #5010: Incorrect status shown for transactions and blocks
### Expected behavior

- Remove confirmations from transaction status (we have finality status now)
- Show valid block and transaction status returned by API

### Actual behavior

- Why are we showing the number of confirmations for the transaction status? Doesn't make any sense because we have the finality now.
- The block status shows final whereas the API returns isFinal to be false.

image

image

image

### Which version(s) does this affect? (Environment, OS, etc...)
3.0

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?

This PR resolves #5010 and #5006

### How was it solved?

- Use correct property name exposed by API

### How was it tested?

Visually and Update unit tests

### CODE REVIEW

File: src/theme/Icon/index.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ import bookmarksIconEmptyStateDark from '@setup/react/assets/images/icons/bookma
 import checkboxFilled from '@setup/react/assets/images/icons/checkmark-filled.svg';
 import checkboxCircleFilled from '@setup/react/assets/images/icons/checkmark-circle-filled.svg';
 import checkmark from '@setup/react/assets/images/icons/checkmark.svg';
+import inprogress from '@setup/react/assets/images/icons/inprogress.svg';
```
Comment (ikem-legend) [inline src/theme/Icon/index.js]: @ManuGowda Please use camel case for icon names

**Review by ikem-legend — COMMENTED** [review]
Looks good. Just a minor change

**Review by ikem-legend — COMMENTED** [review]
Looks good. Just a minor change

**Review by eniolam1000752 — CHANGES_REQUESTED** [review]
Sort by latest block height for a network node seems to be bugged.

When I sort in ascending order, at the bottom, I see that `Unknown` comes before a block height which is lesser that the height just above the `Unknown` value

image

**Review by oskarleonard — DISMISSED** [review]

**Review by eniolam1000752 — DISMISSED** [review]

**Review by ikem-legend — DISMISSED** [review]

**Review by ikem-legend — APPROVED** [review]

**Review by eniolam1000752 — APPROVED** [review]

**Pull request merged**